_data/harnesses/console-consumer-producer-avro/kafka.yml

```diff
@@ -33,8 +33,14 @@ dev:
 
     - title: Create a schema
       content:
-          - action: execute
-            file:
+          - change_directory: console-consumer-producer-avro
```

Comment:
This was the fix that was needed to get the test running on semaphore.  The rest of the changes in the PR are what was needed to get the test-harness running in general.